cmd/vc-rest/startcmd/configuration.go

```diff
@@ -105,16 +104,12 @@ func createStoreProviders(parameters *startupParameters) (*vcStorageProviders, e
 	return &edgeServiceProvs, nil
 }
 
-func createMainStoreProvider(parameters *startupParameters) (vcsstorage.Provider, error) { //nolint: dupl
+func createMainStoreProvider(parameters *startupParameters) (ariesapi.Provider, error) { //nolint: dupl
```

Comment:
Rename it to something like createUtilityStoreProvider, because now it is used only for ld.Context store

Reply:
done.